src/Sensors.cpp

```diff
@@ -306,9 +308,10 @@ bool Sensors::CO2SCD30Read() {
         DEBUG("-->[SCD30] read > done!");
         return true;
     }
-    else {
-        return false;
-    }
+    CO2 = 0;
+    CO2humi = 0;
+    CO2temp = 0;
```

Comment:
I think that is wrong, because you are cleaning variables that maybe are using others sensors? it is i2c?

Reply:
No, you can only read one CO2 sensor. This routine is designed because when the sensor shut down because low battery, the ESP32 continues working and send the last data measured, like if the sensor still work.